src/itensor.jl

```diff
@@ -41,7 +41,7 @@ end
 # ITensor
 #
 
-function sqrt_hermitian(ρ::ITensor; cutoff::Float64 = 1e-15)
+function sqrt_hermitian(ρ::ITensor; cutoff::Float64 = 1e-10)
```

Comment:
Probably we should have a relative tolerance option here, like in Julia's [`isapprox`](https://docs.julialang.org/en/v1/base/math/#Base.isapprox) function.

Reply:
Oh sorry I thought this parameter was for checking hermiticity.

I think the `cutoff` value from `eigen` is already a relative cutoff. I guess you were seeing larger negative eigenvalues?

Reply:
Correct

Reply:
Maybe it is better to specify the cutoff in the case where you were seeing an issue, instead of changing the default value. I would prefer to have a stricter default value.